src/libstd/panic.rs

```diff
@@ -285,6 +286,12 @@ impl RefUnwindSafe for atomic::AtomicBool {}
 #[stable(feature = "unwind_safe_atomic_refs", since = "1.14.0")]
 impl<T> RefUnwindSafe for atomic::AtomicPtr<T> {}
 
+// https://github.com/rust-lang/rust/issues/62301
+#[stable(feature = "hashbrown", since = "1.36.0")]
+impl<K, V, S> UnwindSafe for collections::HashMap<K, V, S> where K: UnwindSafe, V: UnwindSafe {}
+#[stable(feature = "hashbrown", since = "1.36.0")]
+impl<T, S> UnwindSafe for collections::HashSet<T, S> where T: UnwindSafe {}
```

Comment:
I would expect the set's impl to be automatic from the map's, but that deserves a test.

Reply:
Oh, I was expected `std::collections::HashSet` to be based on `hashbrown::HashSet` but it’s not. It’s based on `std::collections::HashMap`, so you’re right.